### PR TITLE
(feat) Enhancement on navigation button style

### PR DESCRIPTION
**Pull request by icrc-jofrancisco**
# Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
In this pull request, I've made modifications to the navigation buttons within the form. Currently, the buttons have a "header-like" styling, leading to an oversized font. To enhance consistency across sections and text elements, I've realigned the button style to match that of other components.

## Screenshots

Before:
Pasted Graphic 1

After:
Pasted Graphic

## Other

Thanks,

### CODE REVIEW

**Comment by denniskigen** [comment]
Thanks for the PR, @icrc-jofrancisco. Might you have read this thread https://openmrs.slack.com/archives/CKS32D55G/p1710226081710819? Just adding it before I get to reviewing it

**Comment by icrc-jofrancisco** [comment]
> Thanks for the PR, @icrc-jofrancisco. Might you have read this thread https://openmrs.slack.com/archives/CKS32D55G/p1710226081710819? Just adding it before I get to reviewing it

Thanks for your comment @denniskigen .
I read the comments carefully and I think the outcome of my implementation is aligned with the expected result, specifically the resizing of the font. There was also a suggestion to change the color of the previous/next labels (also done in this PR).
That said, I think the PR is ready for review. 
Thank you

File: projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss [hunk 1]
```diff
@@ -23,15 +23,15 @@
 
 .nav-link-text {
   margin-top: 0.125rem;
-  white-space: nowrap;
```
Comment (denniskigen) [inline projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss:26]: I think it's important to keep this around for visual consistency. Having the page name truncated when the titles are too long isn't necessarily a bad trade-off when the full page names are listed on the left panel imo.

**Review by denniskigen — APPROVED** [review]
Thanks, @icrc-jofrancisco. I've left a suggestion. Let's see how this looks and evaluate from there.

File: projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss [hunk 2]
```diff
@@ -23,15 +23,15 @@
...
   overflow: hidden;
   text-overflow: ellipsis;
   max-width: calc(100% - #{spacing.$spacing-02});
-  @include type.type-style('heading-03');
+  @include type.type-style('label-01');
```
Comment (denniskigen) [inline projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss:29]: Looking through this again and `label-01` seems too small for me, @icrc-jofrancisco:

CleanShot 2024-03-28 at 1  01 19@2x

File: projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss [hunk 3]
```diff
@@ -23,15 +23,15 @@
...
 }
 
 .nav-label {
   @include type.type-style('label-01');
+  color: #696969;
```
Comment (denniskigen) [inline projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss:34]: #525252 is the default carbon hex for input labels. It's probably a better fit here for consistency:

CleanShot 2024-03-28 at 1  03 43@2x

Also, when we migrate the form engine to use [carbon colour tokens](https://carbon-elements.netlify.app/colors/examples/preview/), we could swap it out for `gray-70`.

It also gets a better contrast score than #696969:

CleanShot 2024-03-28 at 1  06 37@2x

CleanShot 2024-03-28 at 1  08 04@2x

File: projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss (hunk 2 above)
Comment (denniskigen) [reply projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss:29]: How about `body-01`?

CleanShot 2024-03-28 at 1  10 45@2x

File: projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss (hunk 1 above)
Comment (icrc-jofrancisco) [reply projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss]: Agreed. Thanks.

File: projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss (hunk 2 above)
Comment (icrc-jofrancisco) [reply projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss]: Agreed. Thanks.

File: projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss (hunk 3 above)
Comment (icrc-jofrancisco) [reply projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss]: Agreed. Thanks. @denniskigen

**Comment by denniskigen** [comment]
Thanks, @icrc-jofrancisco!

**Comment by denniskigen** [comment]
Follow-up to say this looks great to me in prod. Thanks for working on it, @icrc-jofrancisco!

**Pull request merged**